components/alert/alert.css

```diff
@@ -20,19 +20,19 @@ body {
   border-radius: 3px;
 }
 
-.clr-gp {
+.color-gp {
   background-color: var(--green-pastel);
 }
 
-.clr-rp {
+.color-rp {
   background-color: var(--red-pastel);
 }
 
-.clr-yp {
+.color-yp {
   background-color: var(--yellow-pastel);
 }
 
-.clr-bp {
+.color-bp {
   background-color: var(--blue-pastel);
```

Comment:
what is bp, yp ,rp ?

Reply:
rp as in red pastel ,yellow pastel and so on i'hv added color to the class name that signifies it's a color, these colors are also defined in my variables so wherever they'll be used i will use the same naming convention.